config/initializers/session_store.rb

```diff
@@ -1,4 +1,4 @@
-if !Rails.env.development? && !Rails.env.production?
+if ENV['RAILS_USE_MEMORY_STORE'] || (!Rails.env.development? && !Rails.env.production?)
```

Comment:
You know, we could say just don't load it while compiling assets.

Think @skateman is creating an environment just for this